projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 11.4-a.3 - 2022-09-13
+### Enhancements
+- VideoPress block (beta): register a video-chapters beta feature with basic controls. [#26177]
+
+### Improved compatibility
+- Jetpack: update styling for Jetpack logo shown in pre-publish panels for Jetpack and Jetpack Social plugins. [#26101, #26162]
+- Pay with PayPal block: the `postLinkUrl` attribute now uses the immutable post GUID to prevent the editor from becoming dirty after publishing a post. [#26063]
```

Comment:
```suggestion
- Pay with PayPal block: the `postLinkUrl` attribute now uses the immutable post GUID to prevent the editor from becoming dirty after publishing a post. [#26063]
```

Wonder if there's a better way of saying `dirty` - I know generally what it means, I think, but can we be more specific?